ocs_ci/ocs/resources/pvc.py

```diff
@@ -122,17 +124,37 @@ def reclaim_policy(self):
             'persistentVolumeReclaimPolicy'
         )
 
+    @retry(UnexpectedBehaviour, tries=5, delay=2, backoff=1)
```

Comment:
Hi @humblec I probably missed the conversation but why we do retry here?

Reply:
for the resize operation to happen.

Reply:
done. updated the patchset.